import-automation/executor/run_import.sh

```diff
@@ -265,6 +267,7 @@ function get_import_config {
 
   # Create an import config file based on default configs.
   # Drop any references to local files for cloud jobs
+ options="gcs_project_id:$GCP_PROJECT storage_prod_bucket_name:$GCS_BUCKET spanner_project_id:$GCP_PROJECT spanner_instance_id:$SPANNER_INSTANCE spanner_database_id:$SPANNER_DB"
```

Comment:
Pls append to existing options from $1 and also align to previous row
`options=$options gcs_project_id:$GCP_PROJECT storage_prod_bucket_name:$GCS_BUCKET spanner_project_id:$GCP_PROJECT spanner_instance_id:$SPANNER_INSTANCE spanner_database_id:$SPANNER_DB"`

Reply:
sorry, i see below that all args have been removed. lets also delete the lines 265,266 to initialize options, config_file